README.md

```diff
@@ -33,6 +33,8 @@ More information on the Mozilla Hacks blog:
 
 - **duplicate** - The aim of this classifier is to detect duplicate bugs.
 
+- **needsdiagnosis** - The aim of this classifier is to detect issues that are likely invalid and don't need to be diagnosed for webcompat usecase.
```

Comment:
I've also put together a document describing the whole process https://github.com/webcompat/webcompat.com/blob/main/docs/ml-process.md. @marco-c  wonder if it's useful to add a link to it here ?

Reply:
Yes, I think that'd be useful!

Reply:
Ok, added the link :)